docs/geedocs/answer/3499964.html

```diff
@@ -18,23 +17,15 @@
 <h1><a href="../index.html">Google Earth Enterprise Documentation Home</a> | Install GEE Server</h1>
 <h2>Uninstalling</h2>
 
-<p>This section provides instructions for uninstalling all or some of the software provided in the Google Earth Enterprise software package. For this procedure, you use the same installer you used to install the software.</p>
-<p><strong>Note</strong>: As with installation, you can use either the command-line uninstaller or the graphical user interface (GUI) uninstaller to remove the Google Earth Enterprise Fusion and Server products.</p>
-<p>To uninstall all or part of the Google Earth Enterprise software:</p>
-<p>The uninstall script starts. Follow the onscreen instructions to remove the Google Earth Enterprise Fusion or Server products.</p>
+<p>GEE provides a command line uninstall script.  To uninstall Google Earth Enterprise Server:</p>
 <ol>
 <li>Open a terminal window on your Linux workstation.</li>
 <li>Log in as <code>root</code>.</li>
 <li>Stop the Google Earth Enterprise Server, if you have a previous version installed and it is running:
-
 <p><code>/etc/init.d/geserver stop</code></p>
 </li>
-<li>Navigate to the <code>/opt/install/GoogleEarthServer_Uninstaller/</code> directory.
-<p>To uninstall using the Google Earth Enterprise Server DVD, insert the DVD and mount the drive. The uninstallers listed below are available at the top level of the DVD media.</p>
-</li>
-<li>To uninstall Google Earth Enterprise Server:
-<code>
-./Uninstall_Google_Earth_Enterprise_Server_GUI.sh // GUI uninstaller
-./Uninstall_Google_Earth_Enterprise_Server.sh // Command line uninstaller</code></li></ol></div>
-<p><a href="#top">Back to top</a></p>
-</body> </html>
+<li>Navigate to the location at which you initially cloned the GEE git repo.</li>
+<li>Within the GEE git repo, navigate to <code>earthenterprise/earth_enterprise/src/installer</code>
```

Comment:
Does the installer not install the uninstall script under `/opt/google`? It may not be for this task, but it seems that if we're using an uninstall script, it should come along with the whole installation, and not require the repository clone.